wf-galleon/WFLY-19021-Stability_In_Provisioning.adoc

```diff
@@ -0,0 +1,252 @@
+---
+categories:
+  - wf-galleon
```

Comment:
@jfdenise This needs to be category defined in https://github.com/wildfly/wildfly-proposals/blob/main/_data/wildfly-categories.yaml.

Please have a chat with @jmesnil about what category we should use for this. Unless I missed something, it looks like we need a new one. But if we do, let's pick one that's both reasonably intuitive and can fit with likely future proposals.

Reply:
@bstansberry , it seems that it already exists: https://github.com/wildfly/wildfly-proposals/blob/main/_data/wildfly-categories.yaml#L136

Reply:
oops; sorry for the noise!